LiveWallpaperQuotes/app/src/main/res/xml/meta.xml

```diff
@@ -3,5 +3,4 @@
     xmlns:android="http://schemas.android.com/apk/res/android"
     android:author="@string/wallpaper_author"
     android:description="@string/wallpaper_description"
-    android:settingsActivity="com.stanleyidesis.quotograph.ui.activity.LWQActivateActivity"
```

Comment:
- [ ] Not sure this was the _best_ move, but I think it helps clarify the issue with the inability to modify settings before finishing the FTUE